app/src/main/java/org/simple/clinic/enterotp/BruteForceOtpEntryProtectionConfigModule.kt

```diff
@@ -2,12 +2,13 @@ package org.simple.clinic.enterotp
 
 import dagger.Module
 import dagger.Provides
+import io.reactivex.Observable
 import java.time.Duration
 
 @Module
 class BruteForceOtpEntryProtectionConfigModule {
   @Provides
-  fun config(): BruteForceOtpEntryProtectionConfig {
-    return BruteForceOtpEntryProtectionConfig(limitOfFailedAttempts = 4, blockDuration = Duration.ofMinutes(20))
+  fun config(): Observable<BruteForceOtpEntryProtectionConfig> {
```

Comment:
I don't personally see a reason for this to be `Observable`, since the values are constant for the config. Let me know if I got something wrong here.